src/module/Subject/src/Subject/Manager/SubjectManager.php

```diff
@@ -107,9 +107,9 @@ public function getUnrevisedRevisions(TaxonomyTermInterface $term)
                 }
             }
         }
-        
+
         $collection = new ArrayCollection();
-        $this->iterEntities($filteredEntities, $collection, 'normalize');
+        $this->iterEntities($filteredEntities, $collection, 'isRevised');
```

Comment:
The children get added in this `iterEntities`, (or more specifically in `iterLinks`, called in this method).